### DISCUSSION TITLE
Can I replace postgresql with edgedb easily?

### DISCUSSION THREAD

**Discussion by taowang1993** [geldata/gel#7885] [Q&A]
I am self-hosting Dify on a vps with postgres as the database.
https://github.com/langgenius/dify
I wonder if it is easy to just replace the postgresql service to edgedb in the docker-compose.yaml file.
For example, I can swtich to supabase very easily using the same env variables.
Does it work the same for edgedb?
```
DB_USERNAME=
DB_PASSWORD=
DB_HOST=
DB_PORT=
DB_DATABASE=
```

**Comment by cghiurea** [comment]
No
